README.md

```diff
@@ -88,17 +95,19 @@ finalsetup.yaml
 The work done by each recipe is saved as a `.tar.gz` file. So you would take the last archive, `finalsetup.tar.gz`, to extract on the PineNote. Currently, the archive's size is about 240MB and extracted into the partition would occupy almost 700MB.
 
 ## Install the rootfs on the PineNote
-Basically, you have to extract the `finalsetup.tar.gz` inside the prepared partition. You need to follow Martyn's and Dorian's guides to get to this point.
+Take the `*.tag.gz` file after the latest step and extract it to the prepared partition.
+You need to follow Martyn's and Dorian's guides to get to this point.
 
-Here is how I'm installing the rootfs. On my laptop connected to the PineNote booted in Android, I do:
+For example, let `finalsetup.tar.gz` be the resulting archive.
+Then, to install the rootfs using my laptop connected to the PineNote booted in Android, I do:
 ```
 $ adb push finalsetup.tar.gz /sdcard/Download
 $ adb shell
 $ su
 # mkdir /sdcard/target
 # mount /dev/block/mmcblk0p17 /sdcard/target
 # cd /sdcard/Download
-# tar xzf finalsetup.tar.gz -C /sdcard/target
+# busybox tar xzf finalsetup.tar.gz -C /sdcard/target
```

Comment:
is busybox definitely required for tar here? I don't recall needing it, but I'd trust you ;)